### PR TITLE
Added documentation of new AllowedMediaHosts setting

### PR DESCRIPTION
**Pull request by bergmania**
Pending 11.3 and 10.5 release - **RC Release: March 23th**

This adds documentation of new setting introduced in https://github.com/umbraco/Umbraco-CMS/pull/13900

### CODE REVIEW

File: 10/umbraco-cms/reference/configuration/contentsettings.md [hunk 1]
```diff
@@ -59,6 +60,11 @@ In the root level section, that is those without a seperate sub section like Ima
 
 If greater control is required than available from the above, this setting can be used to store a list of file extensions. If provided, only files with these extensions can be uploaded via the backoffice.
 
+### Allowed media hosts
+
+By default only relative urls are allowed when getting URLs for resized images or thumbnails using the ImagesController.
+If you need absolute urls you will have to add the allowed hosted in this list. The value could be `["umbraco.com", "www.umbraco.com", "our.umbraco.com"]`.
```
Comment (sofietoft) [inline 10/umbraco-cms/reference/configuration/contentsettings.md:66]:
```suggestion
By default, only relative URLs are allowed when getting URLs for resized images or thumbnails using the ImagesController.
If you need absolute URLs you will have to add the allowed hosts to this list. The value could be `["umbraco.com", "www.umbraco.com", "our.umbraco.com"]`.
```

**Review by sofietoft — COMMENTED** [review]
Thanks for the PR @bergmania !

Just a small suggestion - let me know what you think!

File: 10/umbraco-cms/reference/configuration/contentsettings.md (hunk 1 above)
Comment (bergmania) [reply 10/umbraco-cms/reference/configuration/contentsettings.md:66]: All suggestions are find, we just need to ensure its fixed for both v10 and v11 version

Comment (sofietoft) [reply 10/umbraco-cms/reference/configuration/contentsettings.md:66]: Will do 💪

**Pull request merged**